components/pkg-export-helm/src/chart.rs

```diff
@@ -47,7 +47,12 @@ impl<'a> Chart<'a> {
             .value_of("CHART")
             .unwrap_or(&manifest.pkg_ident.name)
             .to_string();
-        let version = matches.value_of("VERSION");
+        let pkg_version = manifest.pkg_ident.version.clone();
+        let version = matches.value_of("VERSION").or(
+            pkg_version.as_ref().map(|s| {
+                s.as_ref()
```

Comment:
Are you taking a reference to a reference here?

Reply:
not exactly. Getting an `&str` from an `&String`.